### ISSUE TITLE
Requested URL not found on this server

### ISSUE THREAD

**Issue by anonymouz-trash** [Zavy86/WikiDocs#175]
### Describe your issue

Sorry, again, after latest update... :no_good: 

I was working on the wiki than watchtower updates the container image and afterwards something broke.
![Bildschirmfoto_20240826_112211](https://github.com/user-attachments/assets/e970047e-96ad-4430-bbcc-38c1896bd361)

The main page gets loaded normally, but on every link on the sidebar it says "Requested URL not found".
![Bildschirmfoto_20240826_112237](https://github.com/user-attachments/assets/b8567496-9fe5-4734-b560-dfd46bf29475)

What I did:
* Cleaned browser cache and website settings of today
* Restart web browser (Firefox if it is important)
* Looking after `/var/log/apache2/error.log` inside the container, no errors there

I didn't changed any files or their structure outside of the container if you may had re-implemented that check in Issue #163.

Anyway, no critics, keep up the excellent work. :1st_place_medal:

### Device and settings

raspberry pi and docker

### Steps to reproduce

* docker compose down
* docker compose up -d  --force-recreate
* docker exec -it wikidocs /bin/sh
* cat /var/log/apache2/error.log

### Extra fields

- [ ] I'd like to work on this issue

**Comment by Zavy86** [comment]
check for htaccess file the new image do not contain it by default but it where created by the setup.
if you have a running version you have to add it manually check for a sample in the readme

**Comment by anonymouz-trash** [comment]
Oh ok, I read about it in the requirements. I'll check it and if its working I'll let you know and close this issue. Thanks a lot.

**Comment by anonymouz-trash** [comment]
Soo....

I created `.htaccess` file with the contents like described in the manual section in README.md.
But, in my docker compose folder structure I have now `datasets` and created a `config`-folder which contains the `.htaccess` file. Afterwards I had to mount the `.htaccess` inside the container in the web server root which is `/var/www/localhost/htdocs`.

It would be very nice to add this line to the example compose.yaml if it's important:

```
volumes:
   - ./datasets:/datasets
   - ./config/.htaccess:/var/www/localhost/htdocs/.htaccess
```
or you link it like with the `datasets` folder.
